### DISCUSSION TITLE
Why does docker-compose up not reflect my code changes?

### DISCUSSION THREAD

**Discussion by oussama-he** [ciur/papermerge#270] [Q&A] [answered]
I'm trying to add the Arabic language to the project, but my changes don't reflect on the project. I can't see them when running the project like this `docker-compose up`.

How did I know this? I add the line `<h1>My modification</h1>` in the` login.html `file in the `account` folder and I visited the `localhost:8000/account/login/` and I can't find my modification.

**Comment by ciur** [comment, marked as answer]
If you want help regarding custom modifications for Papermerge I am happy to discuss with your commercial support contract terms.
